app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value():
-    pass
+    assert 17 == sum(split_integer(17, 4))
 
 
 def test_should_split_into_equal_parts_when_value_is_divisible_by_parts():
-    pass
+    assert [8, 8, 8, 8, 8] == split_integer(40, 5)
 
 
 def test_should_return_part_equals_to_a_value_when_slitting_into_one_part():
-    pass
+    assert 1 == len(split_integer(25, 1))
```

Comment:
when you write tests you need to make them as obvious as possible. You can write: `split_integer(25, 1) == [25]` and that's all. No need to do additional actions (`len`, `count`, and so on). There are no sense is this